pages/homepage/[postId]/index.js

```diff
@@ -134,6 +142,22 @@ export default function Post({post}) {
                         moreText="Please try again."
                     />
                 )}
+                {pinSuccess && (
+                    <ModalAlert
+                        onClick={() => setPinSuccess(false)}
+                        title="Success"
+                        content={'You have successfully pinned a comment'}
+                        moreText="Marked as resolved!"
+                    />
+                )}
+                {unpinSuccess && (
+                    <ModalAlert
+                        onClick={() => setUnpinSuccess(false)}
+                        title="Success"
```

Comment:
Having two states for different but similar success messages is kinda unnecessary. you can conditionally pick the text of the `content` and `moreText`. But, :shrug: